ansible/data/molecule/requirements.txt

```diff
@@ -46,18 +48,21 @@ pycodestyle==2.5.0
 pycparser==2.19
 pyflakes==2.1.1
 PyNaCl==1.3.0
-pytest==4.4.1
+pyparsing==2.4.0
+pytest==4.6.3
 python-dateutil==2.8.0
 python-gilt==1.2.1
-PyYAML==3.13
-requests==2.21.0
+PyYAML==5.1.1
```

Comment:
is it received from pip3 freeze after molecule installation on empty Ubuntu?

Reply:
yes. I created a new container based on empty Ubuntu and I ran "docker build --build-arg aurora_branch=F#SRC-3352_nvidia_container_fix xenial" for this Dockerfile: https://github.com/shadow-robot/aurora/blob/F%23SRC-3352_nvidia_container_fix/docker/molecule/base/xenial/Dockerfile and molecule was installed using requirements.txt in my branch, which includes PyYAML==5.1.1. After molecule installation is complete, pip3 freeze shows "PyYAML==5.1.1", so I know molecule installation has not downgraded PyYaml